sdks/python/apache_beam/transforms/environments.py

```diff
@@ -52,7 +55,7 @@
 from apache_beam.utils import proto_utils
 
 if TYPE_CHECKING:
-  from apache_beam.options.pipeline_options import PortablePipelineOptions
+  from apache_beam.options.pipeline_options import PortableOptions
```

Comment:
It'll be good to actually start enforcing these type checks so they don't go out of date...